app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -34,6 +34,28 @@ def phase
     phase_from_keys 'claimPhaseDates', 'latestPhaseType'
   end
 
+  def tracked_items
+    items = []
+    items_src = object.data['consolidatedTrackedItemsList'] || []
```

Comment:
We have five lists — neverReceivedFromOthersList neverReceivedFromYouList receivedFromOthersList receivedFromYouList stillNeedFromYouList — plus the meta-list of consolidatedTrackedItemsList. Maybe what I should do is to go through the five lists, look at the status of each tracked item (which is an enum of "NEEDED", "SUBMITTED_AWAITING_REVIEW", "INITIAL_REVIEW_COMPLETE", "ACCEPTED", and "NO_LONGER_REQUIRED" according to their schema) and put the needed items in one list and everything else is another list.